src/mca/schizo/ompi/schizo-ompi-cli.rst

```diff
@@ -10,8 +10,133 @@
 
    $HEADER$
 
+General command line options
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The following general command line options are available.
+
+The ``--add-host`` option
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. include:: /prrte-rst-content/cli-add-host.rst
+
+The ``--add-hostfile`` option
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. include:: /prrte-rst-content/cli-add-hostfile.rst
+
+The ``--allow-run-as-root`` option
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. include:: /prrte-rst-content/cli-allow-run-as-root.rst
+
+The ``--bind-to`` option
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. include:: /prrte-rst-content/cli-bind-to.rst
+
+The ``--debug-daemons`` option
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. include:: /prrte-rst-content/cli-debug-daemons.rst
+
+The ``--debug-daemons-file`` option
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. include:: /prrte-rst-content/cli-debug-daemons-file.rst
+
+The ``--display`` option
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. include:: /prrte-rst-content/cli-display.rst
+
+The ``--dvm`` option
+~~~~~~~~~~~~~~~~~~~~
+
+.. include:: /prrte-rst-content/cli-dvm.rst
+
+The ``--forward-signals`` option
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. include:: /prrte-rst-content/cli-forward-signals.rst
+
+The ``--host`` option
+~~~~~~~~~~~~~~~~~~~~~
+
+.. include:: /prrte-rst-content/cli-dash-host.rst
+
+The ``--hostfile`` option
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. include:: /prrte-rst-content/cli-dvm-hostfile.rst
+
+The ``--leave-session-attached`` option
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. include:: /prrte-rst-content/cli-leave-session-attached.rst
+
+The ``--map-by`` option
+~~~~~~~~~~~~~~~~~~~~~~~
+
+.. include:: /prrte-rst-content/cli-map-by.rst
+
+The ``--output`` option
+~~~~~~~~~~~~~~~~~~~~~~~
+
+.. include:: /prrte-rst-content/cli-output.rst
+
+The ``--personality`` option
```

Comment:
I don't think you want to expose this option as you want the personality "locked" to OMPI.